strawberryfields/apps/graph/max_clique.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Maximum clique

```

Comment:
What about renaming the module to `clique`, to make shorter

Reply:
I'm ok with renaming, but need to keep in mind that `clique` is very general whereas `max_clique` is specific to what we do. For example, the clique module of networkx has more functionalities: https://networkx.github.io/documentation/stable/reference/algorithms/clique.html

Reply:
We definitely don't want clashes, so let's indeed leave as is.